tutor/src/models/user/menu.js

```diff
@@ -40,7 +40,7 @@ const ROUTES = {
   },
   analytics: {
     label: 'Research analytics',
-    isAllowed(course) { return get(course, 'isAP'); },
+    isAllowed(course) { return course && ['ap_biology', 'ap_physics'].includes(course.subject.code); },
```

Comment:
Isn't `course.subject` a string, meaning it has no `code` method? Or am I missing something?

Reply:
hm, I had a big long thing typed here pointing out how subject returns course data.  but now I see that my change didn't get included. I wonder why the spec didn't fail 😡    Great catch!